packages/shared-services/src/notification-service.ts

```
@@ -0,0 +1,86 @@
+      id: Math.random().toString(36).substr(2, 9),
+      type,
+      title,
+      message,
+      timestamp: new Date(),
+      read: false
+    };
+
+    const validatedNotification = NotificationSchema.parse(notification);
+    this.notifications.push(validatedNotification);
+    
+    this.listeners.forEach(listener => listener(validatedNotification));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)